Makefile

```diff
@@ -101,12 +101,12 @@ INTEG_TEST_COVERPKG := -coverpkg="$(MODULE_ROOT)/client/...,$(MODULE_ROOT)/commo
 ##### Tools #####
 update-checkers:
 	@printf $(COLOR) "Install/update check tools..."
-	@go install golang.org/x/lint/golint@latest # golint doesn't have releases.
-	@go install golang.org/x/tools/cmd/goimports@v0.1.5
-	@go install honnef.co/go/tools/cmd/staticcheck@v0.2.1
+	@go install golang.org/x/lint/golint@latest
+	@go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
This is what google recommends for `goimports`: https://pkg.go.dev/golang.org/x/tools/cmd/goimports#pkg-overview.